demisto_sdk/commands/common/docker_helper.py

```diff
@@ -159,6 +159,11 @@ def __init__(self):
     def __del__(self):
         del self.tmp_dir_name
 
+    @functools.lru_cache
+    @staticmethod
+    def version():
+        return init_global_docker_client().version()["Version"]
```

Comment:
```suggestion
    def version() -> Version:
        Version(return init_global_docker_client().version()["Version"])
```

return `Version`, more versatile